scripts/github_issue_template.tmpl

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 NXP Corp.
+# SPDX-License-Identifier: Apache-2.0
+'''
+    git hub issue template
+'''
+
+GITHUB_ISSUE_TITLE_TEMPLETE = '''
+tests-ci :${CASE_NAME} : ${ZEPHYR_VERSION}: ${PLATFORM}: test ${RESULT}
```

Comment:
I would skip zephyr's version in a title. It will make the title too long and obscure the issue. The version is given in the body of the report

Reply:
agree, remove it

Reply:
ok, remove it